### ISSUE TITLE
wrong results with cycle index series of graphs

### ISSUE THREAD

**Issue by mantepse** [sagemath/sage#30691]
<div id="comment:0"></div>

I sometimes get wrong results when computing large coefficients in the cycle index series for graphs, and then also for the isotype generating series:

```
sage: F = species.SimpleGraphSpecies().cycle_index_series()
sage: F.isotype_generating_series().coefficients(9)
[1, 1, 2, 4, 11, 34, 156, 1044, 1044]
```

Unfortunately, I cannot quite reproduce this yet.

It always happens if I interrupt the computation and then ask for the coefficients again, but sometimes it happens otherwise, too.

Component: **combinatorics**

Keywords: **species, LazyPowerSeries**

_Issue created by migration from https://trac.sagemath.org/ticket/30691_

**Comment by mantepse** [comment]
Changed keywords from **species** to **species, LazyPowerSeries**

**Comment by mantepse** [comment]
<div id="comment:8" align="right">comment:8</div>

I cannot reproduce this with the new code, so very likely, this can be closed after #34552.

**Comment by fchapoton** [comment]
so, shall we close this one now ?
